docs/overview/whats-new-cpp-docs.md

```diff
@@ -48,6 +48,7 @@ For the latest C++ conformance status, see [C++ conformance improvements in Visu
 - [`assert Macro, _assert, _wassert`](../c-runtime-library/reference/assert-macro-assert-wassert.md) - Clarified assert behavior
 - [`vsnprintf_s, _vsnprintf_s, _vsnprintf_s_l, _vsnwprintf_s, _vsnwprintf_s_l`](../c-runtime-library/reference/vsnprintf-s-vsnprintf-s-vsnprintf-s-l-vsnwprintf-s-vsnwprintf-s-l.md) - Clarified return values
 - [`setlocale, _wsetlocale`](../c-runtime-library/reference/setlocale-wsetlocale.md) - Added C Runtime UTF-8 support info
+- [`C++ AMP Overview`](../parallel/amp/cpp-amp-overview.md) - Added note about C++ AMP deprecation.
```

Comment:
What do you think about pulling this one change out. It's sitting in the section for what's new in VS 16.8  
I won't create a new section for VS 2022 until VS 2022 goes out (otherwise it looks like VS 2022 is released).

Reply:
sure, but we need to remember to add the note to the release notes. It's important people know about the change when upgrading.

Reply:
These aren't the release notes. This should be noted in the release notes, but not here (yet).